lib/routes/cqu/jwc/namespace.ts

```diff
@@ -0,0 +1,7 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: '重庆大学本科教学信息网',
+    url: 'jwc.cqu.edu.cn',
```

Comment:
```suggestion
    name: '重庆大学',
    url: 'cqu.edu.cn',
```
and the file path should be lib/routes/cqu/namespace.ts